### PR TITLE
Add interfaces

### PR DESCRIPTION
**Pull request by KochTobi**
Add the interfaces for the domain logic

### CODE REVIEW

File: sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/datasources/persons/PersonsDbConnector.groovy [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+ * <b><short description></b>
+ *
+ * <p><detailed description></p>
+ *
+ * @since <version tag>
+ */
+class PersonsDbConnector implements PersonDataSource{
+    /**
+     * Returns the email that is found in the system for a user identified with the auth id
+     * Returns an empty string if nothing could be found.
```
Comment (wow-such-code) [inline sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/datasources/persons/PersonsDbConnector.groovy:15]: Do we need this interface and the subscriber table with its own name and e-mail address? Shouldn't all of the subscribers be in Liferay anyway? The way the db is structured we can only search for first and last name now, which could lead to problems.

Comment (KochTobi) [reply sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/datasources/persons/PersonsDbConnector.groovy:15]: The db has an extra field for exactly this authentication provider.

File: sample-tracking-status-overview-domain/src/main/groovy/life/qbic/business/project/subscribe/Subscriber.groovy [hunk 2]
```diff
@@ -0,0 +1,30 @@
+package life.qbic.business.project.subscribe
+
+/**
+ * <b><short description></b>
+ *
+ * <p><detailed description></p>
+ *
+ * @since <version tag>
+ */
+class Subscriber {
```
Comment (wow-such-code) [inline sample-tracking-status-overview-domain/src/main/groovy/life/qbic/business/project/subscribe/Subscriber.groovy:10]: this information could be taken from liferay as well, but I don't have strong feelings about using a class

**Review by wow-such-code — CHANGES_REQUESTED** [review]
I think we need to discuss the subscriber table. In my opinion using the  e-mail address (or name of the person) to fetch another e-mail address is unnecessary and could lead to additional problems.

**Review by wow-such-code — APPROVED** [review]
Okay, I'm convinced!

**Pull request merged**